website/content/docs/api-gateway/consul-api-gateway-install.mdx

```diff
@@ -305,6 +305,48 @@ spec:
 
 </CodeBlockConfig>
 
+To create a route for a `backendRef` in a different namespace, you must also
+create a [ReferencePolicy](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy).
+
+The following example creates a route named `example-route` in namespace `gateway-namespace` with a `backendRef` in namespace `service-namespace`. Traffic is allowed because of the `ReferencePolicy` named `reference-policy` in namespace `service-namespace` which allows traffic from `HTTPRoutes` in `gateway-namespace` to `Services` in `service-namespace`.
```

Comment:
```suggestion
The following example creates a route named `example-route` in namespace `gateway-namespace`. This route has a `backendRef` in namespace `service-namespace`. Traffic is allowed because of the `ReferencePolicy`, named `reference-policy` in namespace `service-namespace`, allows traffic from `HTTPRoutes` in `gateway-namespace` to `Services` in `service-namespace`.
```

Reply:
@mikemorris there's a stray "of" that makes it read incorrectly
> allowed because of the `ReferencePolicy`, ...